.github/workflows/histopathology-pr.yml

```diff
@@ -16,6 +16,7 @@ on:
 env:
   pythonVersion: 3.7
   folder: hi-ml-histopathology
+  module: histopathology
```

Comment:
I'm not sure this is worth defining the constant here, you're using it once? Also, name is misleading, it should be "module_for_coverage_reporting" or something like that.